### DISCUSSION TITLE
Opposite of `g-`

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1689] [Q&A] [answered]
I tried to do something which I thought would be easy, but it wasn't. I know I can open a columns sheet of a sheet and then type `g-` to hide columns in the source sheet. What's the opposite of that? How can I select columns to unhide them?

**Comment by saulpw** [comment]
- `gv` on the source sheet will unhide all columns;
- on the Columns sheet, select any hidden rows with the typical selection commands (I will sometimes go to a hidden column and press `,` on the width column) and then you can e.g. `ge5` to set them all to a small width;
- you can `Shift+U` to undo hiding column operations (like `-` then `Shift+U` should leave it in the same state as before).

**Comment by cool-RR** [comment, marked as answer]
Thanks, but these are workarounds. Is there any reason that there wouldn't be a command `unhide-selected` that'll be the opposite of `hide-selected`? I'm imagining that the fact `gv` is already taken is an issue, but that command could still exist even without a binding. If we did look for a binding, `gV` would be my suggestion.

**Reply by saulpw** [reply]
How about:

```
ColumnsSheet.addCommand('gz-', 'unhide-selected', 'onlySelectedRows.width = None')
```

**Reply by cool-RR** [reply]
It's a little odd that the widths get reset to some arbitrary number, but it's good enough. Thank you.

**Reply by saulpw** [reply]
`None` means auto-size.  Feel free to change it to a specific width if you want.
